core/src/components/textarea/test/basic/index.html

```diff
@@ -68,11 +68,10 @@
             <ion-textarea clear-on-edit="true"></ion-textarea>
           </ion-item>
 
-          <!-- TODO: Re-add auto grow with PR#24205 -->
-          <!-- <ion-item>
+          <ion-item>
             <ion-label color="primary">Autogrow</ion-label>
             <ion-textarea auto-grow="true"></ion-textarea>
-          </ion-item> -->
+          </ion-item>
```

Comment:
Is this needed? Seems like the `autogrow` test covers it, and that'd be one less thing to worry about in the `basic` test.

Reply:
I think we can remove it. It doesn't add any test coverage that isn't already handled by the existing auto grow tests.